erc-yank.el

```diff
@@ -65,7 +65,7 @@
   :group 'erc-yank)
 
 (defun erc-yank (&optional arg)
-  "Yank or make a gist depending on the size of the yanked text."
+  "Yank or make a gist depending on the size of the yanked text (ARG)."
   (interactive "*P")
```

Comment:
That silences the byte-compiler sure enough.
It's also completely useless.

Instead add a second sentence that describes that "the prefix argument ARG" does.

Reply:
done.